files/pyspark-s3.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+
+import os
+os.environ['PYSPARK_SUBMIT_ARGS'] = '--packages com.amazonaws:aws-java-sdk:1.10.34,org.apache.hadoop:hadoop-aws:2.6.0 pyspark-shell'
```

Comment:
Is this a duplicate setting to the one in the Dockerfile?

Reply:
Good catch.  👍